game/lib/challenges/challenge_cell.dart

```diff
@@ -161,8 +168,10 @@ class _ChallengeCellState extends State<ChallengeCell> {
                       mainAxisAlignment: MainAxisAlignment.start,
                       children: [
                         Container(
-                          padding:
-                              EdgeInsets.symmetric(horizontal: 10, vertical: 2),
+                          padding: EdgeInsets.symmetric(
+                            horizontal: MediaQuery.sizeOf(context).width * 0.02,
+                            vertical: MediaQuery.sizeOf(context).height * 0.003,
```

Comment:
Great work transitioning to MediaQuery! To improve efficiency and readability, consider storing [MediaQuery.sizeOf(context).height] in a local variable instead of calling it multiple times throughout the code. This will help make the logic cleaner and easier to maintain, especially if the value needs to be updated or debugged later.